### ISSUE TITLE
Add missing prototype for disabled in CheckboxesWidget.js

### ISSUE THREAD

**Issue by crumblix** [mozilla-services/react-jsonschema-form#413]
### Prerequisites

- [x ] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [ x] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.

### Description

Really minor .. but I saw it so ....

### Steps to Reproduce

1. [First Step]
You won't see any difference in production.  Just possibly avoid a warning message in development.

From my fork:
Commit: c7a7b4a8bee05fefcc703b2e4f09fb41d83a4b21 [c7a7b4a]

Basically add this to the prototypes:
    disabled: PropTypes.bool,

**Comment by n1k0** [comment]
Could you please create a proper PR? Thanks!

**Comment by crumblix** [comment]
Happy to try!  In truth I haven't done one before ... so bear with me!

Thanks!

**Comment by crumblix** [comment]
That tried to send all my changes not just the one I one commit I wanted to send.  Sorry about that.  I'll try again.

**Comment by n1k0** [comment]
Fixed by #416
